### PR TITLE
add helm and e2e test

### LINKED ISSUE #2007: master helm e2e

### PR DESCRIPTION
**Pull request by lut777**
- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

#### What type of this PR
Examples of user facing changes:
- Features

#### Which issue(s) this PR fixes:
Fixes #2007

### CODE REVIEW

File: kubeovn-helm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+dual_stack:
+  POD_CIDR: "10.16.0.0/16,fd00:10:16::/64"
+  POD_GATEWAY: "10.16.0.1,fd00:10:16::1"
+  SVC_CIDR: "10.96.0.0/12,fd00:10:96::/112"
+  JOIN_CIDR: "100.64.0.0/16,fd00:100:64::/64"
+  PINGER_EXTERNAL_ADDRESS: "114.114.114.114,2400:3200::1"
+  PINGER_EXTERNAL_DOMAIN: "google.com"
+
+perform:
```
Comment (oilbeater) [inline kubeovn-helm/values.yaml:74]:
```suggestion
performance:
```

File: kubeovn-helm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  GC_INTERVAL: 360
+  INSPECT_INTERVAL: 20
+
+debug:
+  ENABLE_MIRROR: false
+
+cni_conf:
+  MASTER_NODES: ""
+  CHECK_GATEWAY: true
+  LOGICAL_GATEWAY: false
```
Comment (oilbeater) [inline kubeovn-helm/values.yaml:86]: not belong to cni_conf

File: kubeovn-helm/values.yaml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
```
Comment (oilbeater) [inline kubeovn-helm/values.yaml:156]: not needed

File: kubeovn-helm/values.yaml [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
```
Comment (oilbeater) [inline kubeovn-helm/values.yaml:129]: not needed

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: kubeovn-helm/values.yaml (hunk 1 above)
Comment (lut777) [reply kubeovn-helm/values.yaml]: fixed.

File: kubeovn-helm/values.yaml (hunk 4 above)
Comment (lut777) [reply kubeovn-helm/values.yaml]: deleted.

File: dist/images/install.sh [hunk 5]
```diff
@@ -352,81 +352,6 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
-metadata:
```
Comment (oilbeater) [inline dist/images/install.sh:355]: why delete this?

File: kubeovn-helm/values.yaml [hunk 6]
```diff
@@ -1,70 +1,94 @@
...
+MASTER_NODES: ""
 
 networking:
   # net_stack could be dual_stack, ipv4, ipv6
   net_stack: ipv4
   enable_ssl: false
+  # network type could be geneve or vlan
   network_type: geneve
+  # tunnel type could be geneve, vxlan or stt
+  tunnel_type: geneve
```
Comment (oilbeater) [inline kubeovn-helm/values.yaml:28]: Use all capital or lower case letters

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: Makefile [hunk 7]
```diff
@@ -202,6 +202,29 @@ kind-install: kind-load-image kind-untaint-control-plane
 	ENABLE_SSL=true dist/images/install.sh
 	kubectl describe no
 
+.PHONY: kind-untaint-control-plane
```
[the remark this answers is not in the document]
Comment (lut777) [reply Makefile:205]: deleted

File: dist/images/install.sh (hunk 5 above)
Comment (lut777) [reply dist/images/install.sh:355]: find duplicated

File: kubeovn-helm/values.yaml (hunk 6 above)
Comment (lut777) [reply kubeovn-helm/values.yaml]: fixed.

**Review by oilbeater — APPROVED** [review]

**Pull request merged**